usr/src/lib/libfakekernel/Makefile.com

```diff
@@ -56,10 +56,15 @@ $(LINTLIB) :=	SRCS = $(SRCDIR)/$(LINTSRC)
 CSTD =       $(CSTD_GNU99)
 C99LMODE =      -Xc99=%all
 
+CFLAGS +=	$(CCVERBOSE)
+
 # Note: need our sys includes _before_ ENVCPPFLAGS, proto etc.
+# Also Note: intentionally override CPPFLAGS, not +=
 CPPFLAGS.first += -I../common
+CPPFLAGS= $(CPPFLAGS.first)
```

Comment:
It looks like you're not even using any prior value of CPPFLAGS though.  This seems strangely constructed.

Why not just CPPFLAGS = -I ../common $(CPPFLAGS) -- assuming you want to prepend and not just eliminate other CPPFLAGS values?

Reply:
I must have been running into some problem I've since forgotten about.
I'll have to look into that.